### PR TITLE
Profile settings

### PR DESCRIPTION
**Pull request by PatrOlen**
Forms for updating user's data:
image

image

### CODE REVIEW

**Comment by PatrOlen** [comment]
For all intents and purposes, the password change logic is temporary, before the way passwords are hashed and validated is reworked.

File: src/client/components/Forms/ChangePasswordForm.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+              {error == 1 && "Please correct your credentials!"}
+              {error == 2 && "Your provided wrong current password."}
+            </span>
+          )}
+
+          {/* {result && (
+            <span className={cx(styles.success, styles.profileFormSpan)}>
+              {"Your profile data has been updated."}
+            </span>
+          )} */}
```
Comment (krzotki) [inline src/client/components/Forms/ChangePasswordForm.tsx:142]: remove unused

File: src/client/components/Forms/ModifyProfileForm.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+  const { reload } = useRouter();
+
+  React.useEffect(() => {
+    // @ts-ignore
+    const cryptoalerts = session?.user?.cryptoalerts;
+    // @ts-ignore
+    const newsletters = session?.user?.newsletters;
+    // @ts-ignore
+    const productupdate = session?.user?.productupdate;
```
Comment (krzotki) [inline src/client/components/Forms/ModifyProfileForm.tsx:30]: As discussed

File: src/client/components/Forms/ModifyProfileForm.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+      const response = await fetch("/api/auth/modifyprofile", {
+        method: "post",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({
+          username,
+          email,
+          email_sub,
+        }),
+      });
+      console.log(email_sub);
```
Comment (krzotki) [inline src/client/components/Forms/ModifyProfileForm.tsx:74]: remove log

File: src/client/components/Forms/ModifyProfileForm.tsx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+        <Image
+          src={"/Profile.svg"}
+          width={32}
+          height={32}
+          className={cx("svg-adaptive")}
+          alt={"profile_set"}
+        />
+        <p>{t("profile_form_label")}</p>
+        <Image
+          src={`/chevron_${chevronIconMode}.svg`}
```
Comment (krzotki) [inline src/client/components/Forms/ModifyProfileForm.tsx:99]: Please rename it

File: src/client/components/Forms/ModifyProfileForm.tsx [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+              {error == 2 &&
+                "User with given username or email already exists!"}
+            </span>
+          )}
+
+          {/* {result && (
+            <span className={cx(styles.success, styles.profileFormSpan)}>
+              {"Your profile data has been updated."}
+            </span>
+          )} */}
```
Comment (krzotki) [inline src/client/components/Forms/ModifyProfileForm.tsx:163]: Remove unused

**Review by krzotki — CHANGES_REQUESTED** [review]

**Review by krzotki — APPROVED** [review]
Good job 🥇

**Pull request merged**